core/jazz_services/index.js

```diff
@@ -83,7 +83,10 @@ module.exports.handler = (event, context, cb) => {
         // 1: GET service by id (/services/{service_id})
         if (event.method === 'GET' && service_id) {
             logger.info('GET service by ID : ' + service_id);
-            let servicePermission = event.services[0].policies;
+            let servicePermission =[];
+            if (event.services && event.services.length) {
```

Comment:
should we send service details if no policy details are available?  I am assuming no policy details means the user is not authorized for any services. is my assumption correct?